exercises/17_sum_time.rb

```diff
@@ -0,0 +1,29 @@
+def sum_time(time_str_1, time_str_2)
+  time_arr_1 = time_str_1.split(":").map {|el| el.to_i}
+  time_arr_2 = time_str_2.split(":").map {|el| el.to_i}
+
+  time_1 = Time.new(2021, 5, 1, time_arr_1[0], time_arr_1[1], time_arr_1[2])
```

Comment:
`Time.zone.parse("04:22:33")` will directly convert it, no need to hardcode 2021, 5 ,1